indexing-service/src/main/java/io/druid/indexing/common/index/RealtimeAppenderatorTuningConfig.java

```diff
@@ -37,8 +37,9 @@
 @JsonTypeName("realtime_appenderator")
 public class RealtimeAppenderatorTuningConfig implements TuningConfig, AppenderatorConfig
 {
-  private static final int defaultMaxRowsInMemory = 75000;
+  private static final int defaultMaxRowsInMemory = 1000000;
```

Comment:
This should use `IndexTaskUtils.DEFAULT_MAX_ROWS_IN_MEMORY` too.

Reply:
will change to `TuningConfig.DEFAULT_MAX_BYTES_IN_MEMORY`